RadixWallet/Features/TransactionReviewFeature/TransactionReview+View.swift

```diff
@@ -167,6 +164,30 @@ extension TransactionReview {
 
 		private let shadowColor: Color = .app.gray2.opacity(0.4)
 
+		@ViewBuilder
+		private func header(_ proposingDappMetadata: DappMetadata.Ledger?) -> some SwiftUI.View {
+			VStack(alignment: .leading) {
+				HStack {
```

Comment:
Probably give this spacing:0 and add a leading space to the thumbnail, but inside the if. As it is, you get a 10 point space always, even when thumbnail is nil

Reply:
> As it is, you get a 10 point space always, even when thumbnail is nil

Not sure how this is useful, but added an explicit spacing to the HStack.

Reply:
See my new comment, this was not my suggestion